src/components/Header.js

```diff
@@ -1,15 +1,30 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
 import logo from '../images/quran.png';
+import { getSurah } from '../redux/chapters/chaptersSlice';
 
 function Header() {
+  const dispatch = useDispatch();
+  const [surah, setSurah] = useState('');
+
+  const handleSearch = () => {
+    dispatch(getSurah(surah));
+  };
+
   return (
-    <div className="header">
-      <img src={logo} alt="quran karim" />
-      <h1>
-        Quran Karim
-        <span>(114 surahs)</span>
-      </h1>
-    </div>
+    <>
+      <div className="header">
+        <img src={logo} alt="quran karim" />
+        <h1>
+          Quran Karim
+          <span>(114 surahs)</span>
+        </h1>
+      </div>
+      <div className="header search-container">
+        <input className="search-input" type="text" placeholder="Search surah by its number" value={surah} onChange={(e) => setSurah(e.target.value)} />
```

Comment:
- [ ] It would be better if we fix the filtering functionality, currently, it is not filtering the `Surahs`. As we can see in the screenshot below, I tried filtering `Al-Faatiha` and I was expecting only `Surahs` with the name `Al-Faatiha` but all the other Surahs popped up as we had when we did not filter. Let us kindly fix it
![Screenshot from 2023-04-27 21-45-16](https://user-images.githubusercontent.com/46388113/234997545-a514bff7-a85c-462b-94c4-ea53ccc304d7.png)

Reply:
Hello @addod19. Sorry but as you can read in the placeholder the surahs are filtered by surah number not name

Reply:
So if you want to get suratul Faatiha then you should write 1 in search input

Reply:
> Hello @addod19. Sorry but as you can read in the placeholder the surahs are filtered by surah number not name

@Muhammad0602 , noted. Thanks